app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split(" ")
```

Comment:
You should check if the command string contains exactly three elements and starts with 'mv'. This is important to ensure the command is valid and follows the expected format as per the task requirements.